### ISSUE TITLE
Why there is no loss.backward in training process?

### ISSUE THREAD

**Issue by freshlu11** [mims-harvard/UniTS#13]
I just found one loss.backward() in function: [memory_check.](https://github.com/mims-harvard/UniTS/blob/main/exp/exp_sup.py#L951) 

but the aim of this function is to find the max batch size, am I right?  So where does the backward process occurs?

**Comment by freshlu11** [comment]
Sorry, I have find the update code:  

`            norm_value = scaler(loss*loss_scale, model_optim, clip_grad=max_norm,
                                parameters=self.model.parameters(), create_graph=False, update_grad=((i + 1) % acc_it == 0))`
